### ISSUE TITLE
how to build/install shared libraries with cmake

### ISSUE THREAD

**Issue by do11** [capnproto/capnproto#692]
When I build capnproto with autotools, shared libraries are installed (but not cmake configs). But when I build with cmake, only static libraries are installed. How to make cmake make build shared libraries? (Or what I'm missing)?

**Comment by harrishancock** [comment]
Hi @do11, the issue of not installing CMake package config files is fixed in master (#575), but there hasn't been a new release yet. Also note that it's buggy right now if installed to the default prefix (#679).

If you use CMake, you can pass `-DBUILD_SHARED_LIBS=ON` on the command line to make it build shared libraries instead of static.

Also, the Google group (https://groups.google.com/forum/#!forum/capnproto) is the best place to ask these sorts of questions.

**Comment by do11** [comment]
Thanks! Well, this is not just 'how make install' question, but discrepancy in install methods which could be interpreted almost as a bug (if I remote 'how' and just say there is different results of install in autotools and cmake). Basically, I want to create rpm so result could affect many, and I want to make it suitable for cmake and autotools users. Thus authors opinion is very useful.

**Comment by harrishancock** [comment]
Good point! Indeed this is a bug, so never mind what I said about the Google group. :)
